onboarding/web-infra.md

```diff
@@ -0,0 +1,23 @@
+# Web Infrastructure Onboarding
+
+This document is an outline of the tasks necessary to onboard a new member of the Web Infrastructure team.
+
+Please note these tasks should only be performed after the person's nomination has passed.
+
+## Tasks
+
+Tasks prefixed by `Member:` are for the new member to complete.
+Tasks prefixed by `Web Admin:` are for a member of `@nodejs/web-admins` to complete.
+
+- [ ] Web Admin: Create an issue in [nodejs/web-team](https://github.com/nodejs/web-team) specifically for onboarding the member and include this list in it.
+  - Please link any additional issues or PRs relating to the member's onboarding to this issue for record keeping.
+- [ ] Web Admin: Recommend member have some form of physical 2FA (i.e. Yubikey) or passkey enabled on their GitHub account, and that any SSH or GPG key attached to their GitHub account have passphrases.
```

Comment:
The Node.js Organization requires 2FA, so this isn't something we need to explicitly say.

Reply:
We _do_ need to state this -- the org requires 2FA, this is a recommendation specifically to use a physical factor, which AIUI is not enforced at the org level.